projects/extension/build.py

```diff
@@ -8,37 +8,479 @@
 import sys
 import tempfile
 import textwrap
+from collections import OrderedDict
 from pathlib import Path
 
-HELP = """Available targets:
-- help             displays this message and exits
-- build-install    runs build followed by install
-- install          installs the project
-- install-sql      installs the sql files into the postgres installation
-- install-prior-py installs the extension's python package for prior versions
-- install-py       installs the extension's python package
-- uninstall        uninstalls the project
-- uninstall-sql    removes the sql extension from the postgres installation
-- uninstall-py     removes the extension's python package from the system
-- freeze           updates frozen.txt with hashes of incremental sql files
-- build            alias for build-sql
-- build-sql        constructs the sql files for the extension
-- clean            removes python build artifacts from the src dir
-- clean-sql        removes sql file artifacts from the sql dir
-- clean-py         removes python build artifacts from the extension src dir
-- build-release    runs build-sql and updates the version in __init__.py
-- test             runs the tests in the docker container
-- test-server      runs the test http server in the docker container
-- lint-sql         runs pgspot against the `ai--<this_version>.sql` file
-- lint-py          runs ruff linter against the python source files
-- lint             runs both sql and python linters
-- format-py        runs ruff to check formatting of the python source files
-- reformat-py      runs ruff to update the formatting of the python source files
-- docker-build     builds the dev docker image
-- docker-run       launches a container in docker using the docker image
-- docker-stop      stops the container
-- docker-rm        deletes the dev container
-- run              builds+runs the dev container and installs the extension"""
+
+class BuildPyActions:
```

Comment:
nit: WDYT about naming the class just "Actions"? 

```
        BuildPyActions.install_prior_py()
        BuildPyActions.install_py()
        BuildPyActions.install_sql()
```

would be. which for some reason feels much less intrusive to my brain

```
        Actions.install_prior_py()
        Actions.install_py()
        Actions.install_sql()
```

no biggie though

Reply:
Yes, done!